vm_test.go

```diff
@@ -3625,6 +3625,60 @@ func TestSpread(t *testing.T) {
 		"Runtime Error: wrong number of arguments: want=3, got=2")
 }
 
+func TestRunCompiled(t *testing.T) {
+	s := tengo.NewScript([]byte(`fnMap := {"fun1": func(a) { return a * 2 }}`))
+	c, err := s.Run()
+	require.NoError(t, err)
+
+	cFn := c.Get("fnMap").Map()["fun1"].(*tengo.CompiledFunction)
```

Comment:
We should add a function to encapsulate this so it’s easier to get a CompiledFunction